Mixxx e.V. Articles of association (#8)
mixxxdj/proposals: proposals/2024-06-14_mixxx_e_v.md

## Your Proposal Title

* **Owners:**
  * Daniel Schürmann

* **Implementation Status:** `Not implemented`

* **Related Issues and PRs:**

* **Other docs or links:**
  * https://ev.kde.org/corporate/statutes/
  * https://phabricator.kde.org/T12954

* This design doc shall be used to file our Articles of Assoziation 

## Draft

# Articles of Association - Mixxx e.V.

As a non-profit corporation incorporated under German law, Mixxx e.V. is required to declare its intentions and by-laws in a formal "Articles of Association" document. The original and legally binding version is the German version. The following translation is for information only.

## Clarification of Terminology

> 1. Computer programs considered as free software or "open source software" are put into the public domain free of charge by the originator, allowing others the freedom to use, study, adapt, copy, and improve the program for the common good.

Reviewer: We should define the term "Free Open Source Software" here, not the term "Computer programs"

Reviewer: And we should add, which end users benefit from Free Software, e.g. schools, artists, musician with low budget/income.

2. Developing free software involves research, compilation of principles and concepts, and their testing through programming and testing of free software.

> 3. "Mixxx" is a DJ software developed as free software and available to the public free of charge.

Reviewer: We should elaborate "DJ software" here, until now nowhere is mentioned, that Mixxx has something to do with DJing/music

## 1. Company Name, Registered Office, Financial Year

1. The Association is registered as Mixxx e.V.

> 2. The registered offices are located in Berlin. If no fixed office is established, it will be managed from the residence of the Board member acting as general manager.

Reviewer: Berlin? Is this a placeholder?

Author: It is located in the district of the lawyer that will do the registration for us. We still need one.

3. It is listed under the appropriate register in the District Court’s Register of Associations.
4. The financial year is the calendar year.

## 2. Purpose of the Association

1. The Association’s purpose is explain the basics in digital DJ-ing to beginners and educate software developemen students and other interested in digital sound processing and online colaboration. For this pupose we promote and distribute free and open source DJ software, particularly "Mixxx". This gives the equality of opportunity in starting with dgital DJ-ing in bedrooms, school parties and up to professional applications. In addition it allows free exchange of knowledge, for education, science and research.
2. The articles aim to:
   1. Distribute Mixxx as a free and open source software
   2. Distribute Manuals and Guides
   3. Organize user support and education 

>    4. Mentor students during interneships for learning software developmen.

Reviewer: ```suggestion
   4. Mentor students during interneships for learning software development.
```

Reviewer 2: I suggest we add a pre-commit hook with codespell to this repo :)

>    5. Maintaining the development infrstructur for easy code access

Reviewer: ```suggestion
   5. Maintaining the development infrastructure for easy code access
```

3. Amendments to the association’s purpose may only be made in accordance with § 3 (1).

## 3. Non-profit Association

> 1. The Association’s activities are exclusively non-profit and tax-privileged in accordance with German Tax Regulations. Its activities are for the common good and not for profit-making.

Reviewer: ```suggestion
1. The Association’s activities are exclusively non-profit according the criteria for tax-privileged entities in accordance with German Tax Regulations. Its activities are for the common good and not for profit-making.
```

Reviewer 3: ```suggestion
1. The Association’s activities are exclusively non-profit according to the criteria for tax-privileged entities in accordance with German Tax Regulations. Its activities are for the common good and not for profit.
```

2. Financial means may only be used for purposes stipulated in the Articles of Association. Members are reimbursed for expenses but do not receive direct benefits from the Association’s funds.
3. It is not permitted to favor someone through association expenditures that do not serve the association's purpose or are disproportionately high.

## 4. Types of Membership and Members of the Association

1. Membership is open to natural persons and legal bodies supporting the association's aims. Membership types include:

>    1. Active members: Natural persons actively supporting the Association’s purpose and aims.
>    2. Extraordinary members: Natural persons and legal bodies supporting the Association’s aims but without voting rights.
>    3. Supporting Members: Extraordinary members primarily supporting through financial or material contributions.

Reviewer 2: Let's use "Ordinary Member" (de_DE: "ordentliches Mitglied"), it's less ambiguous. Also, 'ets clarify the difference between 2 and 3:

```suggestion
   1. Ordinary members: Natural persons supporting the Association’s purpose and aims with full voting rights.
   2. Extraordinary members: Natural persons and legal bodies supporting the Association’s aims. These members have the right to participate in the General Assembly, but do not have voting rights.
   3. Supporting Members: Extraordinary members primarily supporting through financial or material contributions, without the right to participate in the General Assembly.
```

> 2. Active membership is granted upon suggestion and support by active members or a majority vote. Criteria include long-term commitment and contributions.

Reviewer: ```suggestion
2. Active membership is granted upon suggestion and support by active members and a majority vote. Criteria include long-term commitment and contributions.
```

Reviewer 2: I agree that this is nicer than doing it by board decision. But does this mean we may not grant membership without a general assembly meeting? (that would be okay for me, just want to check I understood this correctly)

Reviewer: This is not mentioned here, at other places Online Vote of the Active Members is explicitly mentioned.

> 3. Membership terminates due to exit, expulsion, death, or loss of legal status. Written requests for termination must be submitted at least 4 weeks prior to December 31st.

Reviewer 2: > Written requests for termination must be submitted at least 4 weeks prior to December 31st.

Can people only leave by the end of the year? Otherwise I don't think this rule makes sense. What do you think about this:
```suggestion
3. Membership terminates due to withdrawal, expulsion, death, or loss of legal status. Withdrawal from the association shall take effect at the end of the following calendar month in which the declaration of withdrawal was submitted.
```

> 4. Expulsion may occur due to serious violations or non-payment of fees over 12 months. Members must be given the opportunity to justify their actions before expulsion.

Reviewer: This process needs to described:

- Who decides about Expulsion
- Is there an immediate Expulsion possible (we can't wait for the yearly General Assembly meeting if a member acts against Mixxx)

Author: I think this should be the same procedure as the 2. Both votes should be possible on a short notice. But we need to define exact conditions to have a rock solid decision process.

5. Members leaving the Association or if the Association is liquidated do not have a right to reimbursement for contributed assets.

## 5. Organs of the Association

- The General Assembly

> - The Board

Reviewer 2: Just to make sure: We do not want an arbitration comittee (de_DE: "Schiedskomission") that decides about internal conflicts (e.g. when a member is expulsed) and we also do not want an audit comittee (de_DE: "Revisionskomission") that checks the spending of funds by the EB?

Author: It all depends on people which are willing to take the job. For now I have only added the bare minimum. The rest is done by a voting during the General Assembly or an internet vote. 

Feel free to add suitable paragraphs that may lint future conflicts.

Reviewer 2: Not saying we need it, the general assembly might suffice. Just wanted to point it out.

## 6. General Assembly

1. The General Assembly consists of all active members, each with one vote.

> 2. It takes place at least once a year, with invitations sent at least six weeks prior.

Reviewer 2: ```suggestion
2. It takes place at least once a year, with invitations sent to all ordinary and extraordinary members at least six weeks prior.
```

> 3. An extraordinary General Assembly may be called if requested by at least 20% of members.

Reviewer: ```suggestion
3. An extraordinary General Assembly may be called if requested by at least 20% of active members.
```

4. The General Assembly has a quorum if the invitation was submitted in proper form.
5. A ¾ majority of votes is required to amend the Articles of Association or decide the liquidation of the Association.

> 6. Active members unable to attend may be represented by another member with written authorization.

Reviewer: ```suggestion
6. Active members unable to attend may be represented by another active member with written authorization.
```

Reviewer 3: whats the criteria for a member to be considered active?

Reviewer 2: It's defined in 4.1.1, but I suggest that we call it "ordinary member" instead to avoid this kind of ambiguity and confusion.

Reviewer 2: @JoergAtGithub I don't understand your suggestion. If you're an active member and cannot participate, why should you name another active member to participate on your behalf? That member may already participate on its own, so it would be a bit pointless. Or would this member then have 2 votes?

Is there a specific reason why an active member is not allowed to name an extraordinary/supporting member as stand-in?

Reviewer: Yes, than this active member would have two votes.

I want to ensure, that we can not loose control over the e.V. - a project like Mixxx will not work, if the end users vote what the developers have to do.

> 

Reviewer 2: I think this is one of the key aspects that should be added. We have potential members from all over the world. As much as I would like to meet in person with all of you, I guess it's kind of unrealistic to make this happen. Hence I would be nice if follow a hybrid approach as a general rule to allow participation from all members, even if they lack the time/money/passport to travel to the meeting location.

```suggestion
7. Participation and voting in the General Meeting must also be possible via electronic communication. By resolution of the Executive Board, a General Meeting may be held exclusively by these means.

```

I took https://www-vereinswiki-info.translate.goog/node/244?_x_tr_sl=auto&_x_tr_tl=de&_x_tr_hl=de&_x_tr_pto=wapp as inspiration, but I'd rather make it a general rule, because if an EB resolution is required, the lack of it might be abused for undemocratic purposes.

## 7. Duties of the General Assembly

1. The General Assembly is the highest decision-making organ and is responsible for all duties unless otherwise assigned.
2. It elects the Board and can deselect board members.
3. It decides on member expulsion objections.
4. It receives the Board’s annual report and auditor’s report, and discharges the Board’s responsibilities.
5. It decides on amendments to the Articles of Association and liquidation of the Association.
6. It presents annual accounts and reports, and appoints auditors.
7. It decides on purchases, sales, investments, and raising loans.

## 8. The Board

> nsists of the Chairman, the Second Chairman, and the Treasurer. 

Reviewer: ```suggestion
Consists of the Chairman, the Second Chairman, and the Treasurer. 
```

1. The Board consists of the Chairman, the Second Chairman, and the Treasurer elected for a 3-year term.
2. The Board’s duties and meeting procedures are stipulated in the by-laws.
3. The Board decides on all matters unless a General Assembly decision is required.
4. Each Board member has the right to represent the Association.
5. Provisional board members are appointed in case of a vacancy until the next General Assembly.
6. The Board can appoint a full-time director to manage current affairs.
7. The director participates in General Assemblies and Board meetings.
8. The Board can amend the Articles of Association for formal reasons as required by authorities.

## 9. By-Laws

> The by-laws govern all details of the Association’s activities not covered in the Articles of Association. The Board implements the by-laws following member approval.

Reviewer: ```suggestion
The by-laws govern all details of the Association’s activities not covered in the Articles of Association. The Board implements the by-laws following active member approval.
```

## 10. Meeting Minutes

All decisions by the Board and General Assembly are documented in writing and available for members’ perusal.

## 11. Labour Agreements

The Board regulates remuneration for full-time employees in the by-laws.

## 13. Financing of the Association

> 1. The Association is funded through membership fees, public contributions, donations, and other financial contributions.

Reviewer 2: "other financial contributions" is kind of handwavy, does this need to be defined further or do you think this suffices?

Also, should we add here that the association is non-profit and any revenue (e.g. from selling merchandise) shall be used in accordance with article 3.2? Or is that clear anyway?

Author: I have no idea. The good thing is that all this has been started from the original KDE.eV Satzung (see git log).
That dos not mean we can't improve it. So go ahead ..

> 2. Membership fees are decided by the General Assembly or through an internet vote.

Reviewer: ```suggestion
2. Membership fees are decided by the General Assembly or through an internet vote of the active members.
```

3. In case of dissolution or liquidation, all assets will go to a suitable non-profit organization for non-profit purposes.

## 14. Inception of the Articles of Association

These Articles of Association came into effect following a General Assembly decision and replace any previous Articles effective since their date of establishment.

> 

Reviewer 4: ## 15. Future of Mixxx
As a commitment to all past and future contributors in the most broad sense as there are coders, analists, designers, translators, writers, testers etc and everyone who ever has put trust in Mixxx, all people taking (future) responsabilities in the Mixxx entity need to cherish and preserve the name, the spirit, philosophy and independence of the entity and it's products.
Not the name nor any part of Mixxx can be sold or commercialised without a written official permission of all previous contributors (broad sense) .
A commercial engagement with a third party can only be granted in order to supply merchandise towards the members, or for occasional activities for members.
Mixxx may never connect it's name to any comercial organisation, it's products may never be used for commercial activities.
Mixxx products may never be abused to violate peoples privacy in any possible way.
Mixxx will avoid getting involved in political movements, 
Mixxx will honour human and animal rights, zero waste policy and will attempt to be carbon neutral.

Reviewer 3: These are all honorable goals, but I don't think they belong here as we can't guarantee to uphold all of them, nor even verify whether we do.

Reviewer 3: >  Not the name nor any part of Mixxx can be sold or commercialised without a written official permission of all previous contributors (broad sense) .
>  A commercial engagement with a third party can only be granted in order to supply merchandise towards the members, or for occasional activities for members.
>  Mixxx may never connect it's name to any comercial organisation, it's products may never be used for commercial activities.

This would require registering a trademark iirc. Certainly possible, just nothing we have done now. See [the article we published](https://mixxx.org/news/2020-05-22-you-dont-need-to-pay-for-mixxx/) for more background.

>  Mixxx products may never be abused to violate peoples privacy in any possible way.

Very broad, almost unenforceable IMO.

>  Mixxx will avoid getting involved in political movements,

define political movements? also we do usually build out a census in the team [and](https://mixxx.org/news/2022-03-29-russians-war/) [take](https://mixxx.org/news/2021-03-26-fsf-statement/) [a stance](https://mixxx.org/news/2020-06-29-black-lives-matter/).

>  Mixxx will honour human and animal rights, zero waste policy and will attempt to be carbon neutral.

Ofcourse, but how far does that extend? Since we're using github and github is owned by microsoft, are we affiliated with all the awful shit microsoft has done? Even [github themselves have blood on their hands](https://www.theverge.com/2019/10/9/20906213/github-ice-microsoft-software-email-contract-immigration-nonprofit-donation)

Reviewer 4: * It's always better to start with written goals in a startup-declaration, even if they are sky high. It's sets the code of conduct to strive for. Changing or adapting something like this later will always involve a discussion.
* Political not in the sense of what is common (war in Ukraine/Gaza; Human rights in China/Iran, capital punishment in US, Japan... ,lgbtqia+ rights) but as in supporting a political party or candidate.
* What can be obvious today can be very strange in some years. Mentioning obvious things as a code of conduct to strive for sets the ideas for successors.
* I know the article and I have read it multiple times, it's part of the reason I'm here and writing this.
* Maybe investigating a trademark deposal could be a task for the new board?
* Violating privacy = strive: not cooperating on privacy violating tools. (brand new future :-) ), like all kind of aps that collect all kind of information to feed the big data monster.
* As we eat and drink we're already part of a lot of shit. It's a strive.

*** The only guarantee is the commitment. Maybe another task for the future board is explaining our conduct in the annual report as a kind of mirror. 
*** I repeat: it's 100 times easier to start with a written code of conduct than ever add or adjust this.
*** Writing this down is also a message to all contributors and users, a kind of fairtrade label.

Reviewer 2: > Not the name nor any part of Mixxx can be sold or commercialised without a written official permission of all previous contributors (broad sense) .
A commercial engagement with a third party can only be granted in order to supply merchandise towards the members, or for occasional activities for members.

This would also mean that we cannot sell Mixxx t-shirts to fund development/hosting without having written permission ftom every past contributor. Since Mixxx is already 20 years old, some might even have passed away by now. I think this is way too strict.

I suppose you want to prevent transferring mixxx to a for-profit company? I agree that this would be nice, but the code is already protected by GPL, so the community may fork. And the trademark would be nice to protect, but I'm unsure if we can enforce that in the statue of the association.

Author: Thank you for this good discussion. I think I understand the base idea of Eve's proposal to state obvious things in the "Articles of association" because in the event that once is bending it we can use the written statements to prevent it. 

The difficult part is, to do it in the right extend. 

The conflict is here that we not have yet the e.V. to transfer all legal stuff from individuals to the e.V. 
It will be an exiting time to go to the process. 

I will try to incorporate the idea in a following commit. Please verify.

Author: I have added a sentence about building an inclusive community and about poitical neutrality. 

> the spirit, philosophy and independence of the entity and it's products.

Since the Mixxx e.V. covers mainly the non-profit education purpose, this does not fit yet. 
Once we have transferred the trademark we may add some sentences about that, on the other hand later changes of the document are expensive because of the legal obligations. We may add such to the by-law? 

> A commercial engagement with a third party can only be granted in order to supply merchandise towards the members, or for occasional activities for members. Mixxx may never connect it's name to any comercial organisation, it's products may never be used for commercial activities.

I would love to have a commercial part that is able to employ a full time worker. These activities shall not against the Terms of the GPL and the terms of this document. So I think this is kind of covered? 

> Mixxx products may never be abused to violate peoples privacy in any possible way.

This should be covered sufficient by 14. and 2.2.6 without mentoring Mixxx itself. 

> Mixxx will honour human and animal rights, zero waste policy and will attempt to be carbon neutral. 

This is important, but hard to archive using GitHub build servers is the opposite of carbon neutral. 
So I would not make it part of our Satzung.

Reviewer 4: I like the inclusive sentence.
Watch out with 'should be covered', depending on laws, general terms and articles that can be changed by third parties. Concrete philophy, ideas on paper can't.

Author: You mean the data protection, right? 
How could we make that more explicit. 

Mixxx e.V. will not publish software that does not follw the EU data protection rules ..

Reviewer 4: Even the EU GDPR is not a bible, a change of coalition in the EP, a new majority and articles can be changed. The discussion about AI in EP is going that way. The first drafts were much better, half of the original ruleshad to leave because of economical aspects.'Yes but it should be possible to ....'
In the end profit came before 'what is right'.
Now's the discussion about anonimity on the internet.
Look what happens in the US, there are Big Tech companies funding a certain candidate because he'would change some laws, another wealthy man leaves California because of the 'inclusive laws'.
Counting on general laws and regulations isn't as good as stipulating what you want yourself.

We could write something as: 
We honour the choice of our contributors and users to participate or use Mixxx in complete anonimity  or with complete identifications on Mixxx websites, discussion fora and development platform.  Who,, how, wherefore  and with whom people use our software is not in our interest. Mixxx honours letter secrecy for all communication between developers and users. 
Mixxx will never gather information about it's users without their knwoledge, fi to offer support.

Author: I have tried to incoperate your suggestions. Please have a look.

   available to the public free of charge. It allows users to mix music tracks 
   in real-time, providing features such as beatmatching, crossfading, and 
   various sound effects. 
   
## 1. Company Name, Registered Office, Financial Year

1. The Association is registered as Mixxx e.V.
2. The registered offices are located in Berlin. If no fixed office is 
   established, it will be managed from the residence of the Board member 
   acting as general manager.
3. It is listed under the appropriate register in the District Court’s 
   Register of Associations.
4. The financial year is the calendar year.

## 2. Purpose of the Association

1. The Association’s purpose is explain the basics in digital DJ-ing to 
   beginners and educate software developemen students and other interested in 
   digital sound processing and online colaboration. For this pupose we 
   promote and distribute free and open source DJ software, particularly 
   "Mixxx". This gives the equality of opportunity in starting with dgital 
   DJ-ing in bedrooms, school parties and up to professional applications. In 
   addition it allows free exchange of knowledge, for education, science and 

>    research. The Association does not suplly the music played with it's 
>    distributed software, the user takes all responsabilities about the rights 
>    of played music.

Reviewer 2: ```suggestion
   research.
```
Remove? I don't think this belongs in this document.

2. It elects the Board and can deselect board members.
3. It decides on member expulsion objections.
4. It receives the Board’s annual report and auditor’s report, and discharges 
   the Board’s responsibilities.
5. It decides on amendments to the Articles of Association and liquidation of 
   the Association.
6. It presents annual accounts and reports, and appoints auditors.
7. It decides on purchases, sales, investments, and raising loans.

## 8. The Board

1. The Board consists of the Chairman, the Second Chairman, and the Treasurer 
   elected for a 3-year term.
   
   TODO: Define the role responsibilities. 
2. The Board’s duties and meeting procedures are stipulated in the by-laws.
3. The Board decides on all matters unless a General Assembly decision is 
   required.
4. Each Board member has the right to represent the Association.
5. Provisional board members are appointed in case of a vacancy until the next 
   General Assembly.
6. The Board can appoint a full-time director to manage current affairs.
7. The director participates in General Assemblies and Board meetings.
8. The Board can amend the Articles of Association for formal reasons as 
   required by authorities.

> 

Reviewer 4: 9. Functions in the organisation do not represent an extra value of power in decision making concerning development. Development decisions have to be democratic, but based on knowledge, research and expertise.

Reviewer 2: I would put in the in purpose of the association that the association should support Mixxx Development, not excise control over it.

Reviewer 4: You need to write the procedure if a boardmember wants to quit / is not anymore able to acts / passes away .... 
+ how + why can boardmembers be fired

Author: I have clarified that

   needs. FOSS is particularly beneficial for schools, artists and musicians
   with low budgets or incomes, as it gives them access to powerful tools for
   education, creativity and productivity without financial barriers.
2. Developing free and open source software involves research, compilation of
   principles and concepts, and their testing through programming and testing
   of free software.
3. "Mixxx" is a DJ software developed as free and open source software and
   available to the public free of charge. It allows users to mix music tracks
   in real-time, providing features such as beatmatching, crossfading, and
   various sound effects.

## 1. Association Name, Registered Office, Financial Year

1. The Association is registered as Mixxx e.V.
2. The registered offices are located in Berlin. If no fixed office is
   established, it will be managed from the residence of the Board member
   acting as general manager.
3. It is listed under the appropriate register in the District Court’s
   Register of Associations.
4. The financial year is the calendar year.

## 2. Purpose of the Association

1. The Association’s purpose is explain the basics in digital DJ-ing to

>    beginners and educate software developemen students and other interested in

Reviewer 4: developemen -> developement

## 4. Types of Membership and Members of the Association

1. Membership is open to natural persons and legal bodies supporting the
   association's aims. Membership types include:
   1. **Core members**: Natural persons actively supporting the Association’s
      purpose and aims. Primarily, they are expected to cooperate and contribute,
      participate in the general assembly and exercise their voting rights.
   2. **Extraordinary members**: Natural persons and legal bodies supporting the
      Association’s aims but without voting rights and obligations to cooperate.
      This gives members the opportunity to pause their cooperation and contribution
      to Mixxx without necessarily losing their membership.
   3. **Supporting Members**: Extraordinary members primarily supporting through
      financial or material contributions. They have the right to participate in
      the General Assemblies without voting rights.
2. Core membership is granted upon suggestion or application by the General
   Assembly or through an internet vote of the core members. Criteria include
   long-term commitment and contributions.

> 3. An core members can change status to that of an extraordinary members by

Reviewer 4: An ?

> # Your Proposal Title

Reviewer 5: please set a title

* **Owners:**
  * Daniel Schürmann

> * **Implementation Status:** `Not implemented`

Reviewer 5: we should have a bullet for core members to officially approve the document (I'd prefer to have it inline rather than just as a PR approval). We can each file a separate signed commit to add our names

```suggestion
* **Implementation Status:** `Not implemented`

* **Signed By:**
```

Author: The German version of this Satzung needs to be signed by at least 7 founding members of Mixxx e.V. I need to check how this can be done electronically. Not sure if a self created certificate is valid for this.

Reviewer: Let's make it easy and invest in 7 post stamps.

* **Related Issues and PRs:**

* **Other docs or links:**
  * <https://ev.kde.org/corporate/statutes/>
  * <https://phabricator.kde.org/T12954>

* This design doc shall be used to file our Articles of Association

## Draft

## Articles of Association - Mixxx e.V. -

As a non-profit corporation incorporated under German law, Mixxx e.V. is
required to declare its intentions and by-laws in a formal "Articles of
Association" document. The original and legally binding version is the German
version. The following translation is for information only.

## Clarification of Terminology

1. Free and Open Source Software (FOSS) refers to computer programs that

>    respects the freedom of the user and the community. It grants everyone the

Reviewer 5: ```suggestion
   respect the freedom of the user and the community. It grants everyone the
```

## 4. Types of Membership and Members of the Association

1. Membership is open to natural persons and legal bodies supporting the
   association's aims. Membership types include:
   1. **Core members**: Natural persons actively supporting the Association’s
      purpose and aims. Primarily, they are expected to cooperate and contribute,
      participate in the general assembly and exercise their voting rights.
   2. **Extraordinary members**: Natural persons and legal bodies supporting the
      Association’s aims but without voting rights and obligations to cooperate.
      This gives members the opportunity to pause their cooperation and contribution
      to Mixxx without necessarily losing their membership.
   3. **Supporting Members**: Extraordinary members primarily supporting through
      financial or material contributions. They have the right to participate in
      the General Assemblies without voting rights.
2. Core membership is granted upon suggestion or application by the General
   Assembly or through an internet vote of the core members. Criteria include
   long-term commitment and contributions.

> 3. A core members can change status to that of an extraordinary members by

Reviewer 5: ```suggestion
3. A core member can change status to that of an extraordinary member by
```

>    submitting their request to the Board. Should an active member not fulfil the
>    above mentioned duties over a period of two consecutive general meetings, the
>    membership will automatically change into an extraordinary membership.

Reviewer 5: ```suggestion
   submitting their request to the Board. Should an active member not fulfill the
   above mentioned duties over a period of two consecutive general meetings, the
   membership will automatically change into an extraordinary membership.
```

4. Membership terminates due to withdrawal, expulsion, death, or loss of legal
   status. Withdrawal from the association shall take effect at the end of the
   following calendar month in which the declaration of withdrawal was
   submitted.
5. The expulsion of a member may be decided by the General Assembly or by
   Internet vote in the event of a critical situation where an amicable solution
   cannot be reached. The following reasons may cause an expulsion:

>    a) Serious violations of regulations in the Articles of Association,
>       the purpose and aim of the Association or the Code of Conduct.
>    b) non-paiment of membership fees over a period of 12 months despite repeated
>       reminders.

Reviewer 5: these subheads are not rendered correctly in markdown.

I don't think we need to mention reminders in part b. We should definitely get a lawyer to look over this whole thing just to tidy up the language.

```suggestion
   a) Serious violations of regulations in the Articles of Association,
      the purpose and aim of the Association or the Code of Conduct.
   b) non-payment of membership fees over a period of 12 months.
```

   provisional board member on the basis of an internet vote. The provisional
   board member remains in office until the next General Assembly takes place.
   The General Assembly decides about the definite succession within the board.
6. In case expulsion of a board member according to §4 (5) it is replaced by a
   provisional board member.
7. The Board can appoint a full-time director to manage current affairs.
8. The director participates in General Assemblies and Board meetings.
9. The Board can amend the Articles of Association for formal reasons as
   required by authorities.

## 9. By-Laws

The by-laws govern all details of the Association’s activities not covered in
the Articles of Association. The Board implements the by-laws following
core member approval by internet vote.

## 10. Meeting Minutes

All decisions by the Board and General Assembly are documented in writing in
German and available for members’ perusal and for the examination of the
resolutions by the registry court.

## 11. Internet vote

The Association may conduct votes via the Internet. An Internet vote is valid
after an announcement via email or messenger, and if a quorum of at least

> 1/3 of the core members participates. The voting period must be at least two

Reviewer 5: 1/3 is too low imho.  I would prefer to put quorum at 50%+1.  being a core member is a significant responsibility and we should all be good about keeping up to date with mixxx business.

Author: 1/3 is the bare minimum required by the German law for such vote. 

This case is significant:
Imagine we need a quick decision, but only 6 of 15 Members respond?  With the 1/3 rule the vote is valid. With a 1/2 rule we need to call for an extra general assembly. Where the same 6 people or even less can decide. 

What is your reason for proposing 1/2? 
I don't really mind, but let's pick the value for a reason.

Reviewer 5: I like the idea of a "majority of a majority" making decisions.  With 15 members and the 1/3 rule, only 3 people could change Mixxx without the approval from the other 12 -- (5 to have a quorum, 3 is a majority)! That seems dangerous if a small group works in secret to pick a busy time when a lot of people aren't around.

With the higher bar, you'd need 5 out of 15 people (8 to reach a quorum, 5 to have a majority)

Author: OK, that makes sense.

# Founding - Mixxx e.V. -

* **Owners:**
  * Daniel Schürmann

* **Implementation Status:** `Not implemented`

* **Related Issues and PRs:**

* **Other docs or links:**
  * <https://ev.kde.org/corporate/statutes/>
  * <https://phabricator.kde.org/T12954>

* This design doc shall be used to file our Articles of Association

> ## German Draft

Reviewer: Can we split English/German in two seperated files?

Author: Yes, we need to consider anyway where we want to publish them along with other Mixxx e.V. affairs.  What is your idea?

Reviewer 2: Separate  subpage of the website

Die englische Übersetzung dient lediglich der Information.

## Erklärung von Begriffen

1. Freie und quelloffene Software (FOSS) bezieht sich auf
   Computerprogramme, die die Freiheit des Benutzers und der Gemeinschaft
   respektieren. Sie gewähren jedem das Recht, die Software auszuführen,
   zu studieren, zu verändern und weiterzugeben. Dies fördert das Gemeinwohl,
   da jeder die Software nutzen und an seine Bedürfnisse anpassen kann. FOSS
   ist besonders vorteilhaft für Schulen, Künstler und Musiker mit geringem
   Budget oder Einkommen, da sie ohne finanzielle Hürden den Zugang zu
   leistungsfähigen Werkzeugen für Bildung, Kreativität und Produktivität
   ermöglicht.
2. Die Entwicklung von freier und quelloffener Software beinhaltet Forschung,
   die Zusammenstellung von Prinzipien und Konzepten und deren Erprobung durch
   Programmieren und Testen von freier Software.
3. „Mixxx“ ist eine DJ-Software, die als freie und quelloffene Software
   entwickelt wurde und die der Öffentlichkeit kostenlos zur Verfügung steht.
   Sie ermöglicht dem Benutzer das Mischen von Musikstücken in Echtzeit
   und bietet Funktionen wie Beatmatching, Crossfading und verschiedene

>    Soundeffekte.

Reviewer: ```suggestion
1. „Mixxx“ ist der Name der DJ-Software, die der Verein als freie und quelloffene Software
   entwickelt und die der jedermann kostenlos zur Verfügung steht.
   Sie ermöglicht Künstlern das kreative Mischen von Musikstücken in Live-Aufführungen auf der Bühne, auf Partys sowie in Radio- und Online-Übertragungen. Mixxx bietet Funktionen wie Beatmatching, Crossfading und vielfältige
   Soundeffekte.
2. „Mixxx e.V.“ ist der Name des Vereins der "Mixxx" entwickelt und u.a. damit die freie DJ-Kultur unterstützt. 
```

We should move this at the beginning of this section, as only "Art and Culture" and "Education and Science" are recognized by the tax authorities. While developing FOSS in general is no reason for tax excemption (Gemeinnützigkeit).

Reviewer 6: > und die der jedermann kostenlos

und kostenlos zur Verfügung stellt.

## 1. Name, Sitz und Geschäftsjahr

1. Der Name des Vereins lautet Mixxx e.V.
2. Der Verein hat seinen Sitz in ... . Sofern keine feste Geschäftsstelle
   eingerichtet ist, folgt die Verwaltung dem Wohnort des jeweiligen
   Vorstandsmitglieds, das die Geschäftsführung wahrnimmt.
3. Er ist im Vereinsregister des Amtsgerichts ... unter der Nummer ...
   eingetragen.

> 4. Geschäftsjahr ist das Kalenderjahr.

Reviewer: ```suggestion
4. Das Geschäftsjahr ist das Kalenderjahr.
```

## 2. Vereinszweck

1. Zweck des Vereins ist die Förderung der Volks- und Berufsbildung sowie

>    Kunst und Kultur. Er bietet Künstlern die Grundbausteine für ihre

Reviewer: ```suggestion
1. Zweck des Vereins ist die Förderung von Kunst und Kultur, sowie die Volks- und Berufsbildung.
 Der Verein bietet Künstlern die Grundbausteine für ihre
```

Author: Zweck is singular ..

   individuelle Audiodarbietung auf freier und quellenoffener Basis und
   berät diese zu den Grundlagen des digitalen DJings, sowie bei der Umsetzung

>    von Erweiterungen die für deren individuellen künstlerischen Stil nötig

Reviewer 6: ```suggestion
   von Erweiterungen, die für deren individuellen künstlerischen Stil nötig
```

   sind. Darüber hinaus werden Software-Entwickler in digitaler Tonverarbeitung
   und Online-Zusammenarbeit weitergebildet.

>    Zu diesem Zweck fördern und vertreibt der Verein freie und quelloffene DJ-

Reviewer 6: ```suggestion
   Zu diesem Zweck fördert und vertreibt der Verein freie und quelloffene DJ-
```

   Software, insbesondere „Mixxx“. Dies gibt Künstlern die gleichen
   Chancen, unabhängig von ihren finanziellen Möglichkeiten mit dem digitalen

>    DJing zu beginnen und ermöglicht Softwareentwickler, Implementierungsdetails

Reviewer 6: ```suggestion
   DJing zu beginnen und ermöglicht Softwareentwicklern, Implementierungsdetails
```

   zu überprüfen und Änderungen einzubringen.
2. Der Satzungszweck wird insbesondere verwirklicht durch:
   1. die Verbreitung von Mixxx als freie und quelloffene Software für Bildung,
      Kunst und kulturelle Zwecke

>    2. das Erstellen von Handbüchern und Anleitungen, um das Lernen und die

Reviewer: ```suggestion
   2. das Erstellen von Dokumentation und Anleitungen, um das Lernen und die
```

      Entwicklung zu unterstützen
   3. die Organisation von Hilfe für Anwender und Bildungsprogrammen
   4. die Begleitung von Studenten während Praktika in der Softwareentwicklung
   5. den Aufbau einer inklusiven und vielfältigen Gemeinschaft, in der
      Mitwirkende aller Hintergründe, Geschlechter und Fähigkeitsstufen

>       willkommen sind.

Reviewer 6: ```suggestion
      willkommen sind
```

   6. die Aufrechterhaltung der Entwicklungsinfrastruktur für niedrige
      Einstiegshürden
   7. die Unterstützung der Entwicklung von Mixxx, ohne Kontrolle über Details
      auszuüben, die unabhänging von den Regeln und Zweckes des Vereins sind.
   8. die Unterstützung der Behörden mit Fachwissen über die DJ-Hardware und

>       Software (z.B. bei Wettbewerbsuntersuchungen)

Reviewer: ```suggestion
      Software (z.B. bei wettbewerbsrechtlichen Untersuchungen)
```

   verwenden. Die Mitglieder erhalten ausschließlich Erstattungen entstandener
   Kosten, aber keine direkten Zuwendungen aus Mitteln des Verein.
3. Niemand darf durch Vereinsausgaben, die dem Vereinszweck fremd sind oder
   durch unverhältnismäßig hohe Vergütungen begünstigt werden.

## 4. Arten der Mitgliedschaft und Mitglieder des Vereins

1. Mitglieder des Vereins können natürliche und juristische Personen werden,
   die die Ziele des Vereins mittragen und unterstützen wollen. Folgende Arten
   von Mitgliedschaften sind möglich:
   1. **Kernmitglieder**: Natürliche Personen, die den Vereinszweck und dessen
      Ziele aktiv unterstützen. Von ihnen wird in erster Linie erwartet, dass
      sie mitarbeiten und einen Beitrag leisten, an der Mitgliederversammlungen
      teilnehmen und ihr Stimmrecht ausüben.
   2. **Außerordentliche Mitglieder**: Natürliche und juristische Personen,
      die Vereinsziele unterstützen, aber kein Stimmrecht und keine
      Mitwirkungspflicht haben. Dies gibt Mitgliedern die Möglichkeit,
      ihre Mitarbeit und Beiträge zu Mixxx zu pausieren, ohne zwangsläufig
      ihre Mitgliedschaft zu verlieren.

>    3. **Fördernde Mitglieder**: Außerordentliche Mitglieder, die durch

Reviewer: Do we really need the 3rd category "Fördernde Mitglieder". If I read this correct it allows to attend the general assembly for a finacial contribution. Do we want this?

Author: This is the KDE model. It would give a reliable long term support and is a base if we plan to employ people. 
I think at the moment it is useful to have this initially in our Satzung because a later change can be dificult and it also protects the core members from regular fees.

Author: @JoergAtGithub Is OK for you or do we have to adjust something?

Reviewer: Why do have 1. and 3. the right to participate the "Mitgliederversammlung, but 2. not?

Author: Ah I see we just need to move the sentence to 2. Than it is valid for both.

Reviewer: Now we have the sentence twice - not wrong, but redundant

>       finanzielle oder Sachbeitrage unterstützen. Sie haben das Möglichkeit

Reviewer 6: ```suggestion
      finanzielle oder Sachbeitrage unterstützen. Sie haben die Möglichkeit,
```

      ohne Stimmrecht an der Mitgliederversammlung teilzunehmen.
2. Die Kernmitgliedschaft wird auf Vorschlag oder Antrag der
   Mitgliederversammlung oder durch eine Internetabstimmung der Kernmitglieder

>    gewährt. Zu den Kriterien gehören langfristiges Engagement und Beiträge.

Reviewer: ```suggestion
   gewährt. Zu den Kriterien gehören langfristiges Engagement sowie Beiträge zum Programmcode von Mixxx.
```
Beiträge could also be read as financial contributions.

Author: Any non financial contribution should be OK.
I change it to Mitarbeit

3. Ein Kernmitglied kann in den Status eines außerordentlichen Mitglieds
   wechseln, indem es einen Antrag an den Vorstand stellt. Bei Nichterfüllung
   der oben angegebenen Pflichten eines aktiven Mitglieds über zwei
   aufeinanderfolgende ordentliche Mitgliederversammlungen ändert sich die

>    Mitgliedschaft automatisch in eine außerordentliche.

Reviewer: ```suggestion
   Mitgliedschaft automatisch in eine außerordentliche. Auf Antrag kann der Vorstand eine Ausnahme von diesem Automatismus gewähren.
```
Otherwise we would be forced to remove a core member in cases like illness etc.

> 4. Die Mitgliedschaft endet durch Austritt, Ausschluss oder Tod, bei Verlust

Reviewer 6: ```suggestion
4. Die Mitgliedschaft endet durch Austritt, Ausschluss oder Tod, sowie bei Verlust
```

   der Rechtspersönlichkeit. Der Austritt aus dem Verein wird zum Ende des
   folgenden Kalendermonats wirksam, in dem die Austrittserklärung
   eingereicht wurde.
5. Der Ausschluss eines Mitglieds kann von der Mitgliederversammlung oder durch

>    eine Internetabstimmung beschlossen werden, wenn eine kritische Situation

Reviewer: ```suggestion
   eine Internetabstimmung der Kernmitglieder beschlossen werden, wenn eine kritische Situation
```

   vorliegt und eine gütliche Lösung nicht erreicht werden kann. Die folgenden
   Gründe können einen Ausschluss bewirken:
   * a) Schwerwiegende Verstöße gegen die Satzungsbestimmungen,

>      den Zweck und die Ziele des Vereins oder dessen Verhaltenskodex.

Reviewer: ```suggestion
     den Zweck oder die Ziele des Vereins oder dessen Verhaltenskodex.
```

   * b) Nichtbezahlen von Mitgliedsbeiträgen über einen Zeitraum von 12 Monaten.
   Dem Mitglied muss vor dem Ausschluss die Möglichkeit gegeben werden, sich zu

>    rechtfertigen

Reviewer: ```suggestion
   rechtfertigen. Betrifft dies ein Kernmitglied, behält dieses bis zum Ausschluss sein Stimmrecht.
   Im Falle von b), kann der Vorstand Zugriffsrechte auf die Infrastruktur des Vereins entziehen und dem Mitglied untersagen im Namen des Vereins zu kommunizieren.
```

   sechs Wochen vorher mit einer vorläufigen Tagesordnung an alle
   Kernmitglieder und außerordentlichen Mitglieder versendet werden muss. Die
   Mitglieder können dem Vorstand bis zwei Wochen vor der Versammlung
   zusätzliche Tagesordnungspunkte einreichen. Der Vorstand veröffentlicht die
   endgültige Tagesordnung online.
3. Eine außerordentliche Mitgliederversammlung kann einberufen werden, wenn dies
   von mindestens 20 % der Kernmitglieder beantragt wird.
4. Die Mitgliederversammlung ist beschlussfähig, wenn die Einladung in
   ordnungsgemäßer Form erfolgt ist. Bei Stimmengleichheit gilt ein Antrag als
   abgelehnt.
5. Zur Änderung der Satzung oder zur Auflösung des Vereins ist eine 3/4-Mehrheit
   der Stimmen erforderlich.
6. Kernmitglieder, die nicht teilnehmen können, können sich durch ein anderes
   Mitglied mit schriftlicher Vollmacht vertreten lassen.
7. Die Teilnahme an der Mitgliederversammlung und die Stimmabgabe müssen auch auf
   elektronischem Wege möglich sein. Durch Beschluss des Vorstandes kann eine
   Mitgliederversammlung ausschließlich auf diesem Wege abgehalten werden.

## 7. Aufgaben der Mitgliederversammlung

> 1. Die Generalversammlung ist das höchste beschlussfassende Organ und ist

Reviewer: ```suggestion
1. Die Mitgliederversammlung ist das höchste beschlussfassende Organ und ist
```
We should use the same term accross the document. Generalversammlung or Jahreshauptversammlung is also fine. Generalversammlung / "General assembly"  might be better understandable in the English translation.

Author: This is the term KDE uses. But I don't mind to change it.

   für alle Aufgaben zuständig, sofern diese nicht anderen zugewiesen
   wurden.
2. Sie wählt den Vorstand aus dem Kreis der Kernmitgliedern und kann
   Vorstandsmitglieder abwählen.
3. Sie entscheidet über Einsprüche gegen den Ausschluss von Mitgliedern.
4. Sie nimmt den Jahresbericht des Vorstandes und den Bericht der
   Rechnungsprüfer entgegen und erteilt dem Vorstand Entlastung.
5. Sie beschließt über Satzungsänderungen und die Auflösung des Vereins.
6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
   schriftlich vorzulegen. Sie beruft einen Rechnungsprüfer, die weder dem
   Vorstand noch einem vom Vorstand berufenen Gremium angehören und auch nicht
   beim Verein angestellt ist, um die Buchführung einschließlich Jahresabschluss
   zu prüfen und über das Ergebnis vor der Mitgliederversammlung zu berichten.
   Dr Rechnungsprüfer hat Zugang zu allen Buchungs- und Rechnungsunterlagen des
   Vereins.
7. Sie beschließt über Käufe, Verkäufe, Investitionen und die Aufnahme von
   Krediten.

## 8. Der Vorstand

1. Der Vorstand besteht aus dem Vorsitzenden, dem zweiten Vorsitzenden und dem

>    Schatzmeister die für eine Amtszeit von 3 Jahren gewählt werden.

Reviewer 6: ```suggestion
   Schatzmeister, die für eine Amtszeit von 3 Jahren gewählt werden.
```

   TODO: Definieren Sie die Verantwortlichkeiten der Rollen.

2. Die Aufgaben des Vorstands und der Ablauf der Sitzungen sind wird durch eine
   Geschäftsordnung des Vorstands geregelt.
3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die Beschlüsse
   der Mitgliederversammlung aus.
4. Jedes Vorstandsmitglied hat das Recht, den Verein zu vertreten.
5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit

>    bestimmt der Vorstand auf Basis einer Internetwahl ein kommissarisches

Reviewer: ```suggestion
   bestimmt der Vorstand ein kommissarisches
```
It doesn't matter, how the remaining two persons communicate.

   Vorstandsmitglied. Das kommissarische Vorstandsmitglied bleibt bis zur
   nächsten Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
   Vorstand entscheidet die Mitgliederversammlung.
6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
   durch ein kommissarisches Vorstandsmitglied ersetzt.

> 7. Der Vorstand kann zur Führung der laufenden Geschäfte einen hauptamtlichen
>    Geschäftsführer bestellen.

Reviewer: ```suggestion
7. Der Vorstand kann zur Führung der laufenden Geschäfte einen
   Geschäftsführer bestellen.
```

8. Der Geschäftsführer nimmt an den Mitgliederversammlungen und
   Vorstandssitzungen teil.

> 9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den

Reviewer 6: ... ohne Mitgliederabstimmung / eigenständig ... ?

Author: Yes, for my understanding this is redundant with the now removed Severability Clause. 
It is only for authorities demand regarding the tax status. Here a formal process of a General Assembly is to long. I will add that the general assembly needs to confirm this change later.

Author: @ronso0 is this OK now?

   Behörden verlangt wird.

## 9. Geschäftsordnung

Die Geschäftsordnung regelt über den Rahmen der Satzung hinausgehende Details
der Vereinsarbeit. Der Vorstand setzt die Geschäftsordnung nach der Zustimmung
der Kernmitglieder in einer Internetabstimmung in Kraft.

## 10. Protokolle

Alle Beschlüsse des Vorstandes und der Mitgliederversammlung werden schriftlich
in deutscher Sprache dokumentiert und stehen den Mitgliedern zur Einsichtnahme

> und zur Prüfung der Beschlüsse durch das Registergericht zur Verfügung.

Reviewer: Is it necessary to document daily board decisions in German language too?

Author: No. only tax relevant things. I think it is more easy to have everything in German. 
Please also not that our daily Mixxx work is to a Beschluss, we need a Beschluss when we spend money and than it needs to be in German anyway.

Reviewer: I don't think, that the board of an e.V. need to do written Beschluss by law, if something has to be paid. This might be required by the Geschäftsordnung of the e.V., but the tax office requires only that the purpose of each payment is documented.

Author: Ok, I don't know the implications. At least from the KDE e.V. I have learned that they keep their protocols in German. Do you have an alternative sentences that works.

Reviewer 3: I agree that we should try to find a way to not exclude non-german-speaking members here.

Author: What is the proposal? My idea was that we discuss everything in English but keep the protocols in German for legal reasons. Everyone can paste them to DeepL and has a translation in their Mother language fast without extra effort put on the Protokollführer.

Reviewer 3: Not sure if the on-the-fly translation is too much for the Protokollführer. I think it should be up to them to do however they're most proficient and if they don't actually speak german we can help them translate afterwards...
Anyways, it'll probably have to be something like you proposed where we have german protocols for legal reasons and then leverage auto translation for everything else.

Reviewer: All documents for the court and tax office must be in German language by law. The authorities would only accept translations by certified translators, which are very expensive. Therefore the original of all these documents should be in German language. Than we can create informal translation to English ourself, or using an online translator.

Mandatory documents for the authority are at least:
-Gründungsdokumente
-Die Satzung
-Änderungen an der Satzung
-Protokoll der Mitgliederversammlung (incl. Vorstandswahlen, Entlastung des Vorstands, Haushaltsplan, Anträge und Beschlüsse)
-Jahresabschluss mit Bilanz, und Gewinn- und Verlustrechnung
-Tätigkeitsbericht (anhand der tatsächlichen Tätigkeiten des Vereins wird über das Fortbestehen der Gemeinnützigkeit entschieden)
-Vermögensaufstellung
-Mitteilungen über die Anschrift des Wohnsitzes der Vorstandsmitglieder an das Registergericht bei Änderungen

But the daily decisions of the board do not need to be in written form by law. But it make sense to do this for transparency to the own members.

Reviewer 3: I didn't know that. Thanks for the clarification. I see our hands are pretty much tied then.

## 11. Internetabstimmung

> Die Vereinigung kann Abstimmungen über das Internet durchführen. Eine

Reviewer 6: ```suggestion
Der Verein kann Abstimmungen über das Internet durchführen. Eine
```

Internetabstimmung ist gültig nach einer Ankündigung per E-Mail oder im
obligatorischen Gruppenchat der Kernmitglieder und wenn ein Quorum von
mindestens 1/2 der Kernmitglieder teilnimmt. Der Abstimmungszeitraum muss
mindestens zwei Wochen nach der Ankündigung betragen, um sicherzustellen,
dass die Mitglieder genügend Zeit haben, ihre Stimme abzugeben.

## 12. Tarifverträge

Vor der Anstellung hauptamtlich Beschäftigter regelt der Vorstand deren
Bezahlung in der Geschäftsordnung.

## 13. Vereinsfinanzierung

1. Der Verein wird durch Mitgliedsbeiträge, öffentliche Zuschüsse, Spenden
   und sonstige Zuwendungen Dritter finanziert.

> 2. Die Höhe der Mitgliedsbeiträge wird durch die Mitgliederversammlung oder
>    durch eine Internetabstimmung der Kernmitglieder entschieden.

Reviewer: ```suggestion
2. Die Höhe der Mitgliedsbeiträge wird durch die Mitgliederversammlung entschieden.
```
I don't think this requires ad-hoc decisions.

Author: OK.

   Zweck zu verarbeiten oder Dritten zugänglich zu machen. Diese Pflicht
   besteht auch über das Ausscheiden der oben genannten Personen aus dem Verein
   hinaus.
3. Der Verein wahrt das Briefgeheimnis für den gesamten Schriftverkehr gemäß
   Artikel 10 des deutschen Grundgesetzes und verpflichtet sich zur Einhaltung
   und verpflichtet sich, soweit anwendbar, zu gleichwertigen Standards bei
   internationaler Kommunikation.
4. Der Verein respektiert die Privatsphäre der Nutzer der angebotenen
   Web-Dienste und der veröffentlichten Software und erlaubt ihnen die anonyme
   Teilnahme oder mit vollständiger Identifizierung.
5. Die Vereinigung verpflichtet sich zur Transparenz und sammelt keine Daten
   der Nutzer ohne deren ausdrückliche Kenntnisnahme und Zustimmung.
6. In Fällen, in denen die Dienste des Vereins von einem Drittanbieter
   bereitgestellt werden die eine separate Benutzeranmeldung erfordern, gelten
   die Datenschutzbestimmungen des Drittanbieters.

## 14. Salvatorische Klausel

Sollten einzelne Bestimmungen dieser Satzung ganz oder teilweise unwirksam oder
undurchführbar sein oder werden, so berührt dies nicht die Gültigkeit der

> Satzung insgesamt.

Reviewer: Isn't this a clause for contracts? I don't think that this make sense in a statue. Where do we have this from?

Author: Some Satzungen you find online have that clause. It is probably redundant with the clause 8.9 where the board is allowed to change the Satzung.

Reviewer: For such legal clauses I recommend to follow the Mustersatzung for the ministry of justice: https://www.bmj.de/SharedDocs/Downloads/DE/Formular/Mustersatzung_eines_Vereins.pdf?__blob=publicationFile&v=4

## 7. Duties of the General Assembly

1. The General Assembly is the highest decision-making organ and is
   responsible for all duties unless otherwise assigned.
2. It elects the Board from a number of core members and can deselect board
   members.
3. It decides on member expulsion objections.
4. It receives the Board’s annual report and auditor’s report, and discharges
   the Board’s responsibilities.
5. It decides on amendments to the Articles of Association and liquidation of
   the Association.
6. Namely the annual accounts and the annual report have to be presented in
   written format to the General Assembly for decision making and discharge
   of the board. It calls on an auditor, who belong neither to the board nor
   any other panel appointed by the board and who is not employed by the
   association, in order to audit the accounting and annual report and
   relate their results to the General Assembly. The auditors have access
   to all of the Association’s accounting documents.
7. It decides on purchases, sales, investments, and raising loans.

## 8. The Board

1. The Board consists of the Chairman, the Second Chairman, and the Treasurer

>    elected for a 3-year term.

Reviewer 4: Something that came up to me this night:
Why is (should) the board (be) limited to 3 people, it can be another impair number, not all board members need to have a defined 'function' or 'title' (yet).
And as Mixxx is global it would be nice to have different continents represented.

Reviewer: An additional board member means a lot of bureaucratic effort. Each of them needs to be personal identified at the court, for the bank accounts etc.
3 is the minimum by law, and what the most small e.V. have.

Reviewer 4: So practical this means that board members will always have to be living in (or close to) Germany?

Author: Not necessarily, but it helps. 

My understanding/goal for Mixxx e.V. is to just have the bare minimum of board members whith the man purpose of keeping the e.V. legaly running. So an important task is to delegate the work an watch that it is in compliance with the Satzung. All the Core members are kind of extended Bord compared to other e.V. s that do not have the split into core and extraordinary members. 

Does that makes sense?

Reviewer 4: Yes it does, but it's more an ad hoc then a future proof solution.
No one can see in the future, I'm not Madame Soleil either.

Reviewer: The future shouldn't be an issue in this regard, because new board members can only be elected by the general assembly. And the general assembly is also the place to amend the articles of association.

# Founding - Mixxx e.V. -

* **Owners:**
  * Daniel Schürmann

* **Implementation Status:** `Not implemented`

* **Related Issues and PRs:**

* **Other docs or links:**
  * <https://ev.kde.org/corporate/statutes/>
  * <https://phabricator.kde.org/T12954>

* This design doc shall be used to file our Articles of Association

## German Draft

## Satzung des - Mixxx e.V. -

Als gemeinnütziger Verein nach deutschem Recht ist der Mixxx e.V.
verpflichtet, seine Absichten und Geschäftsordung in einer Satzung
festzulegen. Dies ist die rechtsverbindliche deutsche Version.
Die englische Übersetzung dient lediglich der Information.

## Erklärung von Begriffen

1. Freie und quelloffene Software (FOSS) bezieht sich auf
   Computerprogramme, die die Freiheit des Benutzers und der Gemeinschaft

>    respektieren. Sie gewähren jedem das das Recht, die Software auszuführen,

Reviewer 4: 2* das?

   die die Ziele des Vereins mittragen und unterstützen wollen. Folgende Arten
   von Mitgliedschaften sind möglich:
   1. **Kernmitglieder**: Natürliche Personen, die den Vereinszweck und dessen
      Ziele aktiv unterstützen. Von ihnen wird in erster Linie erwartet, dass
      sie mitarbeiten und einen Beitrag leisten, an der Mitgliederversammlungen
      teilnehmen und ihr Stimmrecht ausüben.
   2. **Außerordentliche Mitglieder**: Natürliche und juristische Personen,
      die Vereinsziele unterstützen, aber kein Stimmrecht und keine
      Mitwirkungspflicht haben. Dies gibt Mitgliedern die Möglichkeit,
      ihre Mitarbeit und Beiträge zu Mixxx zu pausieren, ohne zwangsläufig
      ihre Mitgliedschaft zu verlieren.
   4. **Fördernde Mitglieder**: Außerordentliche Mitglieder, die durch
      finanzielle oder Sachbeitrage unterstützen. Sie haben das Möglichkeit
      ohne Stimmrecht an der Mitgliederversammlung teilzunehmen.
2. Die Kernmitgliedschaft wird auf Vorschlag oder Antrag der
   Mitgliederversammlung oder durch eine Internetabstimmung der Kernmitglieder
   gewährt. Zu den Kriterien gehören langfristiges Engagement und Beiträge.#

> 3. Ein Kernmitglied kann in den Status eines außerordentlichen Mitglieds

Reviewer 4: "#" zu viele

   wechseln, indem es einen Antrag an den Vorstand stellt.  Bei Nichterfüllung

>    der oben angegebenen Pflichten eines aktiven Mitglieds über zwei

Reviewer 4: dobelter space für "  Bei"

>    gewährt. Zu den Kriterien gehören langfristiges Engagement und Beiträge.

Author: ```suggestion
   gewährt. Zu den Kriterien gehören langfristiges Engagement und Mitarbeit.
```

   Mitglieder können dem Vorstand bis zwei Wochen vor der Versammlung
   zusätzliche Tagesordnungspunkte einreichen. Der Vorstand veröffentlicht die
   endgültige Tagesordnung online.
3. Eine außerordentliche Mitgliederversammlung kann einberufen werden, wenn dies
   von mindestens 20 % der Kernmitglieder beantragt wird.
4. Die Mitgliederversammlung ist beschlussfähig, wenn die Einladung in
   ordnungsgemäßer Form erfolgt ist. Bei Stimmengleichheit gilt ein Antrag als
   abgelehnt.
5. Zur Änderung der Satzung oder zur Auflösung des Vereins ist eine 3/4-Mehrheit
   der Stimmen erforderlich.
6. Kernmitglieder, die nicht teilnehmen können, können sich durch ein anderes
   Mitglied mit schriftlicher Vollmacht vertreten lassen.
7. Die Teilnahme an der Mitgliederversammlung und die Stimmabgabe müssen auch auf
   elektronischem Wege möglich sein. Durch Beschluss des Vorstandes kann eine
   Mitgliederversammlung ausschließlich auf diesem Wege abgehalten werden.

## 7. Aufgaben der Mitgliederversammlung

1. Die Generalversammlung ist das höchste beschlussfassende Organ und ist
   ist für alle Aufgaben zuständig, sofern diese nicht anderen zugewiesen

>    wurden.

Reviewer 4: 2 * "ist"

6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
   schriftlich vorzulegen. Sie beruft einen Rechnungsprüfer, die weder dem
   Vorstand noch einem vom Vorstand berufenen Gremium angehören und auch nicht
   beim Berein angestellt ist, um die Buchführung einschließlich Jahresabschluss
   zu prüfen und über das Ergebnis vor der Mitgliederversammlung zu berichten.
   Dr Rechnungsprüfer hat Zugang zu allen Buchungs- und Rechnungsunterlagen des
   Vereins.
7. Sie beschließt über Käufe, Verkäufe, Investitionen und die Aufnahme von
   Krediten.

## 8. Der Vorstand

1. Der Vorstand besteht aus dem Vorsitzenden, dem zweiten Vorsitzenden und dem
   Schatzmeister die für eine Amtszeit von 3 Jahren gewählt werden.

   TODO: Definieren Sie die Verantwortlichkeiten der Rollen.

2. Die Aufgaben des Vorstands und der Ablauf der Sitzungen sind wird durch eine
   Geschäftsordnung des Vorstands geregelt.
3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die Beschlüsse
   der Mitgliederversammlung aus.

> 4. Jedes Vorstandsmitglied hat das Recht, den Verien zu vertreten.

Reviewer 4: Verien oder Verein?

   nächsten Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
   Vorstand entscheidet die Mitgliederversammlung.
6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
   durch ein kommissarisches Vorstandsmitglied ersetzt.
7. Der Vorstand kann zur Führung der laufenden Geschäfte einen hauptamtlichen
   Geschäftsführer bestellen.
8. Der Geschäftsführer nimmt an den Mitgliederversammlungen und
   Vorstandssitzungen teil.
9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den
   Behörden verlangt wird.

## 9. Geschäftsordnung

Die Geschäftsordnung regelt über den Rahmen der Satzung hinausgehende Details
der Vereinsarbeit. Der Vorstand setzt die Geschäftsordnung nach der Zustimmung
der Kernmitglieder in einer Internetabstimmung in Kraft.

## 10. Protokolle

Alle Beschlüsse des Vorstandes und der Mitgliederversammlung werden schriftlich
in deutscher Sprache dokumentiert und stehen den Mitgliedern zur Einsichtnahme
und zur Prüfung der Beschlüsse durch das Registergericht zur Verfügung.

## 11. Internetabstimmung

Die Vereinigung kann Abstimmungen über das Internet durchführen. Eine

> Internetabstimmung ist gültig nach einer Ankündigung per E-Mail oder Messenger

Reviewer 3: def "messenger"? Zulip? [facebook messenger](https://en.wikipedia.org/wiki/Messenger_(software))?

Author: Currently Zulip, but I mean a service where we can send group messages. 
"obligatorischen Gruppenchat der Kernmitglieder"

Reviewer 3: sounds good. though I'm not sure if "obligatorisch" is the right word...

Author: This should mean that all core members have to join the core stream. 
Do you agree with that?

Reviewer: Can we add a clause like "The general assembly decides about the internet communication system used for online voting"

und wenn ein Quorum von mindestens 1/2 der Kernmitglieder teilnimmt.
Der Abstimmungszeitraum muss mindestens zwei Wochen nach der Ankündigung
betragen, um sicherzustellen, dass die Mitglieder genügend Zeit haben, ihre
Stimme abzugeben.

## 12. Tarifverträge

Vor der Anstellung hauptamtlich Beschäftigter regelt der Vorstand deren
Bezahlung in der Geschäftsordnung.

## 13. Vereinsfinanzierung

> 1. Der Verein wird durch Mitgliedsbeiträge, öffendliche Zuschüsse, Spenden
>    und sonstige Zuwendungen Dritter finanziert.

Reviewer 3: ```suggestion
1. Der Verein wird durch Mitgliedsbeiträge, öffentliche Zuschüsse, Spenden
   und sonstige Zuwendungen Dritter finanziert.
```

> 2. Die Höhe der Mitgliedsbeiträge wird durch die Mitgliederversammlung oder
>    durch eine Internetabstimmung der Kenmitglieder entschieden.

Reviewer 3: ```suggestion
2. Die Höhe der Mitgliedsbeiträge wird durch die Mitgliederversammlung oder
   durch eine Internetabstimmung der Kernmitglieder entschieden.
```

3. Im Falle der Auflösung oder Liquidation geht das gesamte Vermögen an eine
   geeignete gemeinnützige Organisation für gemeinnützige Zwecke.

## 14. Datenschutz im Verein

> 1. Zur Erfüllung der Zwecke und Aufgaben des Vereins werden werden

Reviewer 4: werden werden

   personenbezogene Daten der Mitglieder unter Beachtung der Vorgaben der
   EU-Datenschutz-Grundverordnung (DS-GVO) und des Bundesdatenschutzgesetzes
   (BDSG) verarbeitet.
2. Den Organen des Vereins und sonstigen für den Verein tätigen Personen ist
   es untersagt ohne Zustimmung, personenbezogene Daten oder private
   Nachrichten zu einem anderen als zu dem zur Aufgabenerfüllung gehörenden
   Zweck zu verarbeiten oder oder Dritten zugänglich zu machen. Diese Pflicht
   besteht auch über das Ausscheiden der oben genannten Personen aus dem Verein
   hinaus.
3. Der Verein wahrt das Briefgeheimnis für den gesamten Schriftverkehr gemäß
   Artikel 10 des deutschen Grundgesetzes und verpflichtet sich zur Einhaltung
   und verpflichtet sich, soweit anwendbar, zu gleichwertigen Standards bei

>    internationaler Kommuniikation.

Reviewer 3: ```suggestion
   internationaler Kommunikation.
```

   with low budgets or incomes, as it gives them access to powerful tools for
   education, creativity and productivity without financial barriers.
2. Developing free and open source software involves research, compilation of
   principles and concepts, and their testing through programming and testing
   of free software.
3. "Mixxx" is a DJ software developed as free and open source software and
   available to the public free of charge. It allows users to mix music tracks
   in real-time, providing features such as beatmatching, crossfading, and
   various sound effects.

## 1. Association Name, Registered Office, Financial Year

1. The Association is registered as Mixxx e.V.
2. The registered offices are located in .... . If no fixed office is
   established, it will be managed from the residence of the Board member
   acting as general manager.
3. It is listed under the appropriate register in the District Court’s
   Register of Associations.
4. The financial year is the calendar year.

## 2. Purpose of the Association

1. The Association’s purpose is to promote education and occupational training
   as well as art and culture. It provides music artists with the basic

>    building blocks for individual performance setups on a free and open source

Reviewer 4: 'music artists'  makes it narrow. There could be video / spoken word /  graphical artists that use Mixxx 
for their performance to.
Maybe just 'artists' ?

Author: I have fiddled quite long with this sentences. We need a not too bulky definition that covers: 
* live DJs
* Beedroom DJs 
* Recorded mix creators.  
* Dance teachers 
* Radio DJs

I would argue that video / spoken word / graphical artists are not at the focus of Mixxx e.V. this does not mean that we actively exclude them, but all things there need are also covered by use cases of the of "music artists ... individual performance" 

The alternative to list the roles explicit covers the risk that it is treated as a definite list, which I like to avoid as well. 

But anyway, I am open to alternative proposals.

Reviewer 4: fi "artists who need reproduction of audio"

Author: artists .. audio performance setups should work

   1. **Core members**: Natural persons actively supporting the Association’s
      purpose and aims. Primarily, they are expected to cooperate and contribute,
      participate in the general assembly and exercise their voting rights.
   2. **Extraordinary members**: Natural persons and legal bodies supporting the
      Association’s aims but without voting rights and obligations to cooperate.
      This gives members the opportunity to pause their cooperation and contribution
      to Mixxx without necessarily losing their membership.
   3. **Supporting Members**: Extraordinary members primarily supporting through
      financial or material contributions. They have the opportunity to participate
      in the General Assemblies without voting rights.
2. Core membership is granted upon suggestion or application by the General
   Assembly or through an internet vote of the core members. Criteria include
   long-term commitment and contributions.
3. A core member can change status to that of an extraordinary member by
   submitting their request to the Board. Should an active member not fulfill the
   above mentioned duties over a period of two consecutive general meetings, the

>    membership will automatically change into an extraordinary membership.

Author: ```suggestion
   membership will automatically change into an extraordinary membership.
   Upon request, the Board may grant an exception to this automatic procedure.
```

4. Membership terminates due to withdrawal, expulsion, death, or loss of legal
   status. Withdrawal from the association shall take effect at the end of the
   following calendar month in which the declaration of withdrawal was
   submitted.
5. The expulsion of a member may be decided by the General Assembly or by

>    an Internet Vote in the event of a critical situation where an amicable solution

Author: ```suggestion
   an Internet Vote of core members in the event of a critical situation where an amicable solution
```

   cannot be reached. The following reasons may cause an expulsion:
   * a) Serious violations of regulations in the Articles of Association,

>      the purpose and aim of the Association or the Code of Conduct.

Author: ```suggestion
     the purpose or aim of the Association or the Code of Conduct.
```

   * b) Non-payment of membership fees over a period of 12 months.
   Members must be given the opportunity to justify their actions
   before expulsion.
6. Members leaving the Association or if the Association is liquidated do not

>    have a right to reimbursement for contributed assets.

Author: ```suggestion
   have a right to reimbursement for contributed assets. If this concerns a core member,
   this member retains their voting rights until they are excluded.
```

   responsible for all duties unless otherwise assigned.
2. It elects the Board from a number of core members and can deselect board
   members.
3. It decides on member expulsion objections.
4. It receives the Board’s annual report and auditor’s report, and discharges
   the Board’s responsibilities.
5. It decides on amendments to the Articles of Association and liquidation of
   the Association.
6. Namely the annual accounts and the annual report have to be presented in
   written format to the General Assembly for decision making and discharge
   of the board. It calls on an auditor, who belong neither to the board nor
   any other panel appointed by the board and who is not employed by the
   association, in order to audit the accounting and annual report and
   relate their results to the General Assembly. The auditors have access
   to all of the Association’s accounting documents.
7. It decides on purchases, sales, investments, and raising loans.

## 8. The Board

1. The Board consists of the Chairman, the Second Chairman, and the Treasurer
   elected for a 3-year term.

   TODO: Define the role responsibilities.

> 

Reviewer 4: Chairman
organisation 
- control values / goals of organisation are honoured 
- organizing meetings
- organizing timeline / schedule
- events?
- represent organisation + PR
- official contacts/engagements fi GSOC
- writing 'status reports'
- organize onMixxx things
- organize offMixxx things
- analog postadress

member functions : 
- welcome new contributors / members
- contact with users (not helpdesk) organizing a newsletter / ezine
- make website 'live', motivate people to write posts
- organizing newsletter / ezine towards the members/users
- help possible discusions being solved
- keep good atmosphere
- intermediate between ...

Vice-Chairman
- tasks from chairman when chairman's absent
- delegated tasks from chairman

Author: Cool thank you this a good start. We need to take care not to overload the Board with task. I think we need also consider the situation where the Mixxx e.V. is running on pilot flame and the Board takes care that it does not goes out. The rest can be put into the by-law.

# Founding - Mixxx e.V. -

* **Owners:**
  * Daniel Schürmann

* **Implementation Status:** `Not implemented`

* **Related Issues and PRs:**

* **Other docs or links:**
  * <https://ev.kde.org/corporate/statutes/>
  * <https://phabricator.kde.org/T12954>

* This design doc shall be used to file our Articles of Association

## German Draft

> ## Satzung des - Mixxx e.V. -

Reviewer 3: trailing `-` intentional?

Author: That's a hack against complains about the trailing . in a headline. 
Is there a better workaround?

Reviewer 3: add `<!-- markdownlint-disable-next-line MD026 -->` above the heading? Alternatively just put Mixxx e.V. in quotes?

Author: OK, Thanks

   4. das Anbieten von Softwareentwicklungspraktika für Studenten, die durch
      erfahrene Vereinsmitglieder (Mentoren) persönlich betreut und unterstützt
      werden.
   5. den Aufbau und die Pflege einer inklusiven und vielfältigen Gemeinschaft,
      die den Vereinszweck unterstützt und über die Vereinsmitglieder hinaus
      Mitwirkende aller Hintergründe, Geschlechter und Fähigkeitsstufen willkommen
      heißt. Der Verein stellt dafür die Kommunikationsplattformen und Werkzeuge
      bereit, um Zusammenarbeit und Kreativität zu fördern und einen einfachen
      Einstieg zu ermöglichen.
   6. die Unterstützung der Entwicklung von Mixxx, ohne Kontrolle über Details
      auszuüben, die unabhänging von den Regeln und Zweckes des Vereins sind.
   7. die Unterstützung der Behörden mit Fachwissen über die DJ-Hardware und
      Software (z.B. bei wettbewerbsrechtlichen Untersuchungen)
3. Eine Änderung des Vereinszwecks darf nur im Rahmen des in §3 (1) gegebenen
   Rahmens erfolgen.

## 3. Gemeinnützigkeit

1. Der Verein verfolgt im Rahmen seiner Tätigkeit gemäß § 2 der Satzung
   ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnittes

>    steuerbegünstigte Zwecke der Abgabenordnung (§§ 51ff. AO). Der Verein ist

Reviewer 3: ```suggestion
   steuerbegünstigte Zwecke der Abgabenordnung (§ 51ff. AO). Der Verein ist
```

Reviewer: > The double §§ is used to reference multiple consecutive paragraphs, which fit's to the use of ff. ("fort folgendende"). I think we want to reference §51 and §52 of the AO. Than §§ and ff. is both correct used here.

Reviewer 3: Didn't know that. thank you.

   auf elektronischem Wege möglich sein. Durch Beschluss des Vorstandes kann
   eine Mitgliederversammlung ausschließlich auf diesem Wege abgehalten werden.

## 7. Aufgaben der Mitgliederversammlung

1. Die Mitgliederversammlung ist das höchste beschlussfassende Organ und ist
   für alle Aufgaben zuständig, sofern diese nicht anderen zugewiesen
   wurden.
2. Sie wählt den Vorstand aus dem Kreis der Kernmitgliedern und kann
   Vorstandsmitglieder abwählen.
3. Sie entscheidet über Einsprüche gegen den Ausschluss von Mitgliedern.
4. Sie nimmt den Jahresbericht des Vorstandes und den Bericht der
   Rechnungsprüfer entgegen und erteilt dem Vorstand Entlastung.
5. Sie beschließt über Satzungsänderungen und die Auflösung des Vereins.
6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
   schriftlich vorzulegen. Sie beruft einen Rechnungsprüfer, die weder dem
   Vorstand noch einem vom Vorstand berufenen Gremium angehören und auch nicht
   beim Verein angestellt ist, um die Buchführung einschließlich
   Jahresabschluss zu prüfen und über das Ergebnis vor der

>    Mitgliederversammlung zu berichten. Dr Rechnungsprüfer hat Zugang zu allen

Reviewer 3: typo
```suggestion
   Mitgliederversammlung zu berichten. Der Rechnungsprüfer hat Zugang zu allen
```

   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die
   Beschlüsse der Mitgliederversammlung aus.
4. Jedes Vorstandsmitglied hat das Recht, den Verein zu vertreten.
5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit
   bestimmt der Vorstand ein kommissarisches Vorstandsmitglied. Das
   kommissarische Vorstandsmitglied bleibt bis zur nächsten
   Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
   Vorstand entscheidet die Mitgliederversammlung.
6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
   durch ein kommissarisches Vorstandsmitglied ersetzt.
7. Der Vorstand kann zur Führung der laufenden Geschäfte einen Geschäftsführer
   bestellen.
8. Der Geschäftsführer nimmt an den Mitgliederversammlungen und
   Vorstandssitzungen teil.
9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den
   Behörden verlangt wird. Über die Änderung wird in der darauffolgenden
   Mitgliederversammlung abgestimmt.

## 9. Geschäftsordnung

Die Geschäftsordnung regelt über den Rahmen der Satzung hinausgehende Details
der Vereinsarbeit. Der Vorstand setzt die Geschäftsordnung nach der Zustimmung

> der Kernmitglieder in einer Internetabstimmung in Kraft.

Reviewer 3: this is oddly specific. This could just be decided by the general assembly, right?

Author: Yes.

Reviewer 3: right, then I think we should reformulate this sentence in terms of that. Otherwise this looks a little weird and suspicious and could become a loophole.

Author: With "in Kraft setzen" is the initial version references. This will happen short after the founding General Assembly and therefore an internet vote is IMHO appropriated. We have not adopted sentences KDE e.V 8.3 about 8.3 Board is able to change the By law if it is done unanimously by vote. It seems to be a common practice to define the mode of change in the By law itself. In Templates you find also versions where a majority of board members is sufficient for a change.  Is think changes of the by law should not required a General assembly to keep the ability to act. We must to make sure that all loophole are removed in this document which is hard to change after we are founded. 

Does that make sense? Do we want to amend this paragraph?

Reviewer 3: I afraid I can't follow you. My initial proposal was just to replace "Kernmitglieder" with "Mitgliederversammlung" because that sounded nicer and afai understand they're equivalent in terms of decision power, but on second thought it doesn't make much sense.

Author: First of all the sentence alone should be understandable. This is probably not yet the case, so please propose a change if you have one in mind. 

In general we have three decision making levels 
* The Board 
* The Internet vote of core members 
* The Gneral Assembly. 

The General Assembly meets once a year, and this  meeting requires quite a bit of formalism. 
The internet vote is a quick thing with the same obligation and the same group of core members. 
The Board are only three people 

In the case of bringing the first by law in force a discussion like in this PR + a vote is probably sufficient. 
= "Kernmitglieder in einer Internetabstimmung"

What we not have discussed is the topics the by-law covers and the mode to iterate it. 
It is the common practice that the Board is able to adjust it on their needs because it may effects their day to day works. We may decide in the by law itself about the core member involvement. If you are afraid of a loophole we may define it here with the risk of hard later changes. 
Fundamental risk for Mixxx or the Association should be covered in this document before the founding, because this is the one that can be only changed with core member involvement and after authorities review.

Reviewer 3: No worries, everything makes sense now.

in deutscher Sprache dokumentiert und stehen den Mitgliedern zur Einsichtnahme
und zur Prüfung der Beschlüsse durch das Registergericht zur Verfügung.

## 11. Internetabstimmung

Der Verein kann Abstimmungen über das Internet durchführen. Eine
Internetabstimmung ist gültig nach einer Ankündigung per E-Mail oder im
obligatorischen Gruppenchat der Kernmitglieder und wenn ein Quorum von
mindestens 1/2 der Kernmitglieder teilnimmt. Der Abstimmungszeitraum muss
mindestens zwei Wochen nach der Ankündigung betragen, um sicherzustellen,
dass die Mitglieder genügend Zeit haben, ihre Stimme abzugeben.

## 12. Tarifverträge

Vor der Anstellung hauptamtlich Beschäftigter regelt der Vorstand deren
Bezahlung in der Geschäftsordnung.

## 13. Vereinsfinanzierung

1. Der Verein wird durch Mitgliedsbeiträge, öffentliche Zuschüsse, Spenden
   und sonstige Zuwendungen Dritter finanziert.
2. Die Höhe der Mitgliedsbeiträge wird durch die Mitgliederversammlung
   entschieden.
3. Bei Auflösung des Vereins oder bei Wegfall steuerbegünstigter Zwecke fällt
   das Vereinsvermögen an den K Desktop Environment e.V. Berlin, der es
   ausschließlich und unmittelbar für gemeinnützige Zwecke im Sinne dieser

>    Satzung zu verwenden hat.

Reviewer 3: KDE is aware of that, right?

Author: Not yet. I want to propose it and if we agree, ask them.

   respect the freedom of the user and the community. It grants everyone the
   right to run, study, modify and redistribute the software. This promotes
   the common good as everyone can use the software and adapt it to their
   needs. FOSS is particularly beneficial for schools, artists and musicians
   with low budgets or incomes, as it gives them access to powerful tools for
   education, creativity and productivity without financial barriers.
4. Developing free and open source software involves research, compilation of
   principles and concepts, and their testing through programming and testing
   of free software.

## 1. Association Name, Registered Office, Financial Year

1. The Association is registered as Mixxx e.V.
2. The registered offices are located in **TODO FILL IN LOCATION** . If no fixed office is
   established, it will be managed from the residence of the Board member
   acting as general manager.
3. It is listed under register number **TODO FILL IN NUMBER** in the **TODO FILL IN LOCATION**
   District Court’s Register of Associations. It is listed under the
   appropriate register in the District Court’s Register of Associations.

> 5. The financial year is the calendar year.

Author: “Name, registered office and financial year” of the association's articles of association, Section 57 (1) BGB states that the association must be entered in the register of associations. Furthermore, an association must have a fixed registered office. If the following sentence only refers to the association's address, this should be defined more precisely: “If no fixed office has been established, the administration shall follow the place of residence of the respective board member who is responsible for the management.

   procedure.
3. Membership terminates due to withdrawal, expulsion, death, or loss of legal
   status. Withdrawal from the association shall take effect at the end of the
   following calendar month in which the declaration of withdrawal was
   submitted.
4. The expulsion of a member may be decided by the General Assembly or by an
   Internet Vote of core members in the event of a critical situation where an
   amicable solution cannot be reached. The following reasons may cause an
   expulsion:
   * a) Serious violations of regulations in the Articles of Association,
     the purpose or aim of the Association or the Code of Conduct.
   * b) Non-payment of membership fees over a period of 12 months.
   Member must be granted a two-week period before expulsion, in which it can 
   justify their actions in writing. In the case of a), the Board may withdraw
   access rights to the Association's infrastructure and the association and
   prohibit the member from communicating on behalf of the association. If this
   concerns a core member, this member retains their voting rights until
   expulsion. 
5. Members leaving the Association or if the Association is liquidated do not

>    have a right to reimbursement for contributed assets.

Author: This does not clearly regulate how the admission procedure works and in what form the declaration of admission should be submitted. Does a future member submit a written application form and then the board, the general meeting etc. decides?

   of the board. It calls on an auditor, who belong neither to the board nor
   any other panel appointed by the board and who is not employed by the
   association, in order to audit the accounting and annual report and
   relate their results to the General Assembly. The auditors have access
   to all of the Association’s accounting documents.
7. It decides on purchases, sales, investments, and raising loans.

## 8. The Board

1. The Board consists of the Chairman, the Second Chairman, and the Treasurer
   elected for a 3-year term.
2. The Board’s duties and meeting procedures are stipulated in the by-laws.
   The Chairman is responsible for following the purpose of the association,
   complying with the Articles of Association and delegation of tasks. The
   second chairperson represents the Chairman in his absence and verifies the
   resolutions of the Board and the General Assembly. The treasurer manages
   the finances of the association, keeps the cash books and the list of
   members.
3. The Board decides on all matters unless a General Assembly decision is
   required. It implements the decisions taken by the General Assembly.

> 4. Each Board member has the right to represent the Association.

Author: The board” it is recommended to increase that each board member may represent individually.

5. Should a board member quit while in office, the board appoints a
   provisional board member. The provisional board member remains in
   office until the next General Assembly takes place. The General
   Assembly decides about the definite succession within the board.
6. The Board can appoint a director to manage current affairs.
7. The director participates in General Assemblies and Board meetings.
8. The Board can amend the Articles of Association for formal reasons as
   required by authorities. The amendment will be voted on at the following
   General Assembly.

## 9. By-Laws

The by-laws govern all details of the Association’s activities not covered in
the Articles of Association. The Board implements the by-laws following
core member approval by internet vote.

## 10. Meeting Minutes

All decisions by the Board and General Assembly are documented in an 
electronically translatable format in German and available for members’ 

> perusal and for the examination of the resolutions by the registry court.

Author: The certification of the resolutions is missing in accordance with § 58 No. 4 BGB. Who signs the minutes of the general meeting?

2. The organs of the Association and other persons working for it are
   prohibited from processing personal data and private messages, or
   making it accessible to third parties, without authorization for purposes
   other than the assigned task. This obligation also applies after the
   above-mentioned persons have left the Association.
3. The Association honors letter secrecy for all communications, in accordance
   with Article 10 of the German Basic Law, and commits to
   equivalent standards for international communications where applicable.
4. The Association respects the privacy choices of users of the provided
   web services and distributed software, allowing them to participate
   anonymously or with full identification.
5. The Association is committed to transparency and will not collect any user
   information without their explicit knowledge and consent.
6. In cases where the Association's services are provided by a third-party
   provider requiring a separate user login, the data protection policies of
   the third-party provider will apply.

## 15. Inception of the Articles of Association

These Articles of Association came into effect following a General Assembly

> decision.

Author: After the founding meeting, the sentence should be amended or supplemented as follows: “These articles of association come into force with the resolution of the general meeting of ***date***.”

   anzuwenden sind, das Bundesreisekostengesetz maßgebend.
3. Es darf keine Person durch Ausgaben, die dem Vereinszweck fremd sind oder
   durch unverhältnismäßig hohe Vergütungen begünstigt werden.

## 4. Arten der Mitgliedschaft und Mitglieder des Vereins

1. Mitglieder des Vereins können natürliche und juristische Personen werden,
   die die Ziele des Vereins mittragen und unterstützen wollen. Folgende Arten
   von Mitgliedschaften sind möglich:
   1. **Kernmitglieder**: Natürliche Personen, die den Vereinszweck und dessen
      Ziele aktiv unterstützen. Von ihnen wird in erster Linie erwartet, dass
      sie mitarbeiten und einen Beitrag leisten, an der Mitgliederversammlungen
      teilnehmen und ihr Stimmrecht ausüben.
   2. **Außerordentliche Mitglieder**: Natürliche und juristische Personen,
      die Vereinsziele unterstützen und an der Mitgliederversammlung teilnehmen
      dürfen, aber kein Stimmrecht und keine Mitwirkungspflicht haben. Dies
      gibt Mitgliedern die Möglichkeit, ihre Mitarbeit und Beiträge zu Mixxx zu
      pausieren, ohne zwangsläufig ihre Mitgliedschaft zu verlieren.
   4. **Fördernde Mitglieder**: Außerordentliche Mitglieder, die durch

>       finanzielle oder Sachbeitrage unterstützen. Sie haben die Möglichkeit

Reviewer 3: numbering broke here

2. Die Aufgaben des Vorstands und der Ablauf der Sitzungen wird durch eine
   Geschäftsordnung des Vorstands geregelt. Der Vorsitzende ist dabei
   für die Verfolgung des Vereinszwecks, die Einhaltung der Satzung und die
   Delegation von Aufgaben verantwortlich. Der zweite Vorsitzende vertritt den
   Vorsitzenden bei dessen Abwesenheit und überprüft die Beschlüsse des
   Vorstands und der Mitgliederversammlungen. Der Schatzmeister verwaltet die
   Finanzen des Vereins, führt die Kassenbücher und die Mitgliederliste.
3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die
   Beschlüsse der Mitgliederversammlung aus.
4. Jedes Vorstandsmitglied hat das Recht, den Verein zu vertreten.
5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit
   bestimmt der Vorstand ein kommissarisches Vorstandsmitglied. Das
   kommissarische Vorstandsmitglied bleibt bis zur nächsten
   Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
   Vorstand entscheidet die Mitgliederversammlung.
6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses

>    durch ein kommissarisches Vorstandsmitglied ersetzt.

Reviewer: Ich glaube nicht, dass es rechtlich möglich ist, ein amtierendes Vorstandsmitglied aus dem Verein auszuschließen. Dies kann eigentlich nur die Mitgliederversammlung. Die kann man natürlich außerordentlich einberufen.

https://www.experto.de/praxistipps/vereinsausschluss-eines-vorstandsmitglieds-so-gehen-sie-vor.html

Author: Ich gehe davon aus das alles in Ordnung ist, da der Ausschluss nur durch die Kernmitglieder bestimmt werden kann, die auch für die Wahl des Vorstands zuständig sind. Oder hast Du einen Änderungsvorschlag?

Reviewer: Die im verlinken Artikel erwähnte BGH-Rechstsprechung besagt, dass nur das gleiche Gremium, dass ein Vorstandsmitglied ernennt, es auch wieder abberufen kann. Also in unserem Fall nur die Mitgliederversammlung.

Der Fall kann also von Gesetzes wegen nicht vorkommen. Daher bitte 6. komplett streichen.

Reviewer 4: I needed to translate this in several translation apps to get the correct meaning, but I suppose it's correct that the organ that gives a mandate can take the mandate back, and only that organ.

Reviewer 3: sorry for this conversation only being in german, but its about german legal text so unless you can read the legal text in question precisely, it doesn't make much sense to participate in the conversation anyways.

Reviewer 4: kein problem, mein Deutsch ist mehr als gut genug, aber bestimmte begriffe sind etwas undeutlich. Ich habe keinen deutschen juridischen hintergrund. Aber viele sachen sind das selbsten in Deutschland wie in Belgien oder Frankreich. So ich verstehe die kerngedachte sehr gut. :-)

Author: OK my interpretation was that the same Group of people have decide in our case they can decide in an internet vote or the general assembly. I am hesitating to delete that paragraph without replacement, because it is also written in the linked article that the Satzung applies and the court jumps in in unclear cases. 

Let's assume there is a twist with a board member that is doing harm to the association and quick solution is required to stop that. 

According to the current Satzung we can withdraw them form the association with §4 (5) by an internet vote which takes two weeks and after the person had the chance to rectify their behaviour.

The alternative would be a extraordinary General Assembly this requires an internet vote to decide for it (2 weeks) and a 6 weeks invitation period. Which is a long time for doing more harm.  
 
What do you think? What suits to us?

Reviewer: You can not withdraw a board member from an e.V. without a general assembly. This is not allowed by law and can not be overruled by the Satzung.

Reviewer 4: We can add an article about the board itself, they're unpair (3) , if 2 of them decide that the third of them is harming the roganization they can take over his responsabilities right away. The board contains the elected representatives from the members, so they can act fast, but this needs to be confirmed with a general assembly's decision in 2 weeks.
If you add an article that the boards decides united and undevided, such a situation is blocked when there's no majority in the board.

Reviewer: No, this is not allowed by German law. Only the general assembly can do this.

Reviewer 4: it's not to withdraw a member of the board it's to block possible harmful begavious/decisions

Reviewer: According to the law:
1. You are board member until the court changes the register - and they do this only based on the protocol of a general assembly
2. A majority vote of the board is needed for decisions of the board
3. Each board member is allowed to speak in the name of the e.V. in public - you can not restrict this

Reviewer 4: 'Each board member is allowed to speak in the name of the e.V. in public ' is not the same as 'Each board member can make commitments or engagements in the name of the e.V'
In the bylaw document can be stipulated that an oral commitment of one of the board members has only value when it's confirmed by a written declaration (digitally) signed by all board members.

Reviewer 4: To continue on Daniels use case: one board member having strange thoughts can't harm the organization that much.
2 or 3 stepping of the right is a complete different dangerous situation.

Author: Ok, this also confirms: 
https://www.iww.de/vb/archiv/vereinsrecht-kein-ausschluss-von-vorstandsmitgliedern-durch-vorstand-f17976#:~:text=093187)-,Vorstandsmitglieder%20k%C3%B6nnen%20nicht%20durch%20den%20Vorstand%20vom%20Verein%20ausgeschlossen%20werden,f%C3%BCr%20den%20Vereinsausschluss%20zust%C3%A4ndig%20ist.

I will delete the 6. 

I will also refine the right to speak up. It shall be written and by a deadline of 2 weeks. OK?

9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den
   Behörden verlangt wird. Über die Änderung wird in der darauffolgenden
   Mitgliederversammlung abgestimmt.

## 9. Geschäftsordnung

Die Geschäftsordnung regelt über den Rahmen der Satzung hinausgehende Details
der Vereinsarbeit. Der Vorstand setzt die Geschäftsordnung nach der Zustimmung
der Kernmitglieder in einer Internetabstimmung in Kraft.

## 10. Protokolle

Alle Beschlüsse des Vorstandes und der Mitgliederversammlung werden in einem 
elektronisch übersetzbaren Format in deutscher Sprache dokumentiert und 
stehen den Mitgliedern zur Einsichtnahme und zur Prüfung der Beschlüsse 
durch das Registergericht zur Verfügung. 

## 11. Internetabstimmung

Der Verein kann Abstimmungen über das Internet durchführen. Eine
Internetabstimmung ist gültig nach einer Ankündigung per E-Mail oder im
obligatorischen Gruppenchat der Kernmitglieder und wenn ein Quorum von
mindestens 1/2 der Kernmitglieder teilnimmt. Der Abstimmungszeitraum muss
mindestens zwei Wochen nach der Ankündigung betragen, um sicherzustellen,
dass die Mitglieder genügend Zeit haben, ihre Stimme abzugeben.

## 12. Tarifverträge

> Vor der Anstellung hauptamtlich Beschäftigter regelt der Vorstand deren

Reviewer: ```suggestion
Vor der Anstellung Beschäftigter regelt der Vorstand deren
```

* **Other docs or links:**
  * <https://ev.kde.org/corporate/statutes/>
  * <https://phabricator.kde.org/T12954>

* This design doc shall be used to file our Articles of Association

## German Draft

<!-- markdownlint-disable-next-line MD026 -->
## Satzung des - Mixxx e.V.

Als gemeinnütziger Verein nach deutschem Recht ist der Mixxx e.V.
verpflichtet, seine Absichten und Geschäftsordnung in einer Satzung
festzulegen. Dies ist die rechtsverbindliche deutsche Version.
Die englische Übersetzung dient lediglich der Information.

## Erklärung von Begriffen

1. „Mixxx“ ist der Name der freien und quelloffenen DJ-Software, die der Verein
   ohne Lizenzkosten zur Verfügung stellt. Sie ermöglicht Künstler*innen das
   kreative Mischen von Musikstücken in Live-Aufführungen auf der Bühne, auf
   Partys und Tanzkursen sowie in Radio- und Online-Übertragungen. Mixxx bietet
   Funktionen wie Beatmatching, Crossfading und vielfältige Soundeffekte.
2. „Mixxx e.V.“ ist der Name des Vereins der "Mixxx" entwickelt und u.a. damit
   die freie DJ-Kultur unterstützt.
3. Freie und quelloffene Software (FOSS) bezieht sich auf
   Computerprogramme, die die Freiheit der Benutzer*innen und der Gemeinschaft

>    respektieren. Sie gewähren jedem das Recht, die Software auszuführen,

Reviewer 4: I completely support the idea of making the text gender neutral  but is the introduction of the 
"*innen" not just making it more limited and more difficult to read?
Wouldn't it be more clear if we add a footnote that stipulates that all demonstrative pronouns and all personal descriptive nouns need to be read in function of all possible genders?
Just add a footnote at the first occurence with a reference to Article 3.5 'Build and maintain an inclusive and diverse community.. '

Author: That has been discussed in Germany a lot and there is still a discussion in progress. However at least for science papers some rules have been established where the using gener neutral alternatives if not possible * are preferred. For my feeling at least for Job titles the gender * has already established. The rest is still cumbersome.

You proposal has been dismissed and will lead to a lower score in Exams. 

https://www.scribbr.de/wissenschaftliches-schreiben/gendern-in-wissenschaftlichen-arbeiten/

I must admit that I am also not too fond of the result. I have edited the text with the help of my daughter. She is more into it.

Reviewer 4: OK, Gendern in wissenschaftlichen Arbeiten: So formulierst du richtig! 😄 
Dan mussen wir sicherstellen, das es nach Deutschen wissenschaftlichen und administratieve standards korrekt ist.

Reviewer 3: > You proposal has been dismissed and will lead to a lower score in Exams.

Well, if you're in bavaria its the other way around because conservatives like starting culture wars
https://www.dw.com/en/germanys-bavaria-sets-limits-on-gender-inclusive-language/a-68618217

Anyways, as someone whose significantly younger than both of you two, I very much appreciate the progressive move!

Reviewer 4: Well, you have a name containing 'b0y' so that wil always make you younger 😄 
I'm 21 forever!

Add: All caused by the influence of Mr Pumpkinhead ... 4 years halloween

   Nachrichten zu einem anderen als zu dem zur Aufgabenerfüllung gehörenden
   Zweck zu verarbeiten oder Dritten zugänglich zu machen. Diese Pflicht
   besteht auch über das Ausscheiden der oben genannten Personen aus dem Verein
   hinaus.
3. Der Verein wahrt das Briefgeheimnis für den gesamten Schriftverkehr gemäß
   Artikel 10 des en Grundgesetzes und verpflichtet sich zur Einhaltung
   und verpflichtet sich, soweit anwendbar, zu gleichwertigen Standards bei
   internationaler Kommunikation.
4. Der Verein respektiert die Privatsphäre der Nutzer der angebotenen
   Web-Dienste und der veröffentlichten Software und erlaubt ihnen die anonyme
   Teilnahme oder mit vollständiger Identifizierung.
5. Die Vereinigung verpflichtet sich zur Transparenz und sammelt keine Daten
   der Nutzer ohne deren ausdrückliche Kenntnisnahme und Zustimmung.
6. In Fällen, in denen die Dienste des Vereins von einem Drittanbieter
   bereitgestellt werden die eine separate Benutzeranmeldung erfordern, gelten
   die Datenschutzbestimmungen des Drittanbieters.

## 15. Inkrafttreten

Diese Satzung tritt in Kraft mit Beschlussfassung der Mitgliederversammlung vom

> **DATUM**

Reviewer 5: Need to put in the date

> **DATUM**

Reviewer 4: 5/03/2025

Unterschriften:

> [Handschriftliche Unterschrift]

Reviewer 5: need to add names

[Name in Druckschrift]

> 

Reviewer 4: Antoine Colombier
Nikolaus Einhauser
Jan Holthuis
Ronny Sandig
Daniel Schürmann
Evelynne Veys
Jörg Wartenberg
Owen Williams

Reviewer 5: @daschuer can you add these names to the PR?

2. The organs of the Association and other persons working for it are
   prohibited from processing personal data and private messages, or
   making it accessible to third parties, without authorization for purposes
   other than the assigned task. This obligation also applies after the
   above-mentioned persons have left the Association.
3. The Association honors letter secrecy for all communications, in accordance
   with Article 10 of the German Basic Law, and commits to
   equivalent standards for international communications where applicable.
4. The Association respects the privacy choices of users of the provided
   web services and distributed software, allowing them to participate
   anonymously or with full identification.
5. The Association is committed to transparency and will not collect any user
   information without their explicit knowledge and consent.
6. In cases where the Association's services are provided by a third-party
   provider requiring a separate user login, the data protection policies of
   the third-party provider will apply.

## 15. Inception of the Articles of Association

These Articles of Association came into effect following a General Assembly

> decision from **Data**

Reviewer 5: need date and names

Reviewer 4: 5/03/2025

Antoine Colombier
Nikolaus Einhauser
Jan Holthuis
Ronny Sandig
Daniel Schürmann
Evelynne Veys
Jörg Wartenberg
Owen Williams

## 2. Purpose of the Association

1. The Association’s purpose is to promote education and occupational training
   as well as art and culture. It provides live performance artists with the
   basic building blocks for individual music-performing setups on a free and
   open source basis and advises them on the basics of digital DJ-ing as well as
   the implementation of unique extensions necessary for their personal

>    artistic style. It also also educate software developers in digital sound

Reviewer 4: educateS

   processing and online collaboration.
   For these purposes, we promote and distribute free and open source DJ

>    software, particularly "Mixxx". This gives artists equal of opportunities

Reviewer 4: 'of' can be deleted

## 2. Vereinszweck

1. Zweck des Vereins ist die Förderung von Kunst und Kultur, sowie die Volks-
   und Berufsbildung. Der Verein bietet Künstler*innen die Grundbausteine für 
   ihre individuelle Audiodarbietung auf freier und quellenoffener Basis und
   berät diese zu den Grundlagen des digitalen DJings, sowie bei der Umsetzung
   von Erweiterungen, die für deren individuellen künstlerischen Stil nötig
   sind. Darüber hinaus werden Softwareentwickler*innen in digitaler
   Tonverarbeitung und Online-Zusammenarbeit weitergebildet.
   Zu diesem Zweck fördert und vertreibt der Verein freie und quelloffene DJ-
   Software, insbesondere „Mixxx“. Dies gibt Künstler*innen die gleichen 
   Chancen, unabhängig von ihren finanziellen Möglichkeiten mit dem digitalen 

>    DJing zu beginnen und ermöglicht Softwareentwickler*innnen, 

Reviewer 3: ```suggestion
   DJing zu beginnen und ermöglicht Softwareentwickler*innen, 
```

   5. den Aufbau einer inklusiven und vielfältigen Gemeinschaft, in der
      Mitwirkende aller Hintergründe, Geschlechter und Fähigkeitsstufen
      willkommen sind
   6. die Aufrechterhaltung ein Entwicklungsinfrastruktur für niedrige

>       Einstiegshürden hinsichtlich Soft- und Hardware.

Reviewer: Why this addional limitation "hinsichtlich Soft- und Hardware". The previous wording covered all other infrastructure like Github, Transifex, GMail etc. too. Let's keep this wording open.

Author: It was my failed try to acknowledge Eve's idea and clarify the sharing controllers is also the Association purpose. Do you have an alternative idea to mention it in a way without limiting us by details.

Reviewer: I think the old text allowed everything.

And such a controller sharing needs a good concept, that the e.V. doesn't loose the non-profit state. Therefore we must neither generate to much income, nor should we give monetary value (free controllers) to individuals. For me this is an additional complex topic, which we shouldn't define in the Satzung. With an open text, we can define this anytime later, when our Schatzmeister became familar with all these tax related topics.

Author: OK, I will revert to the old version.

   3. **Fördernde Mitglieder**: Außerordentliche Mitglieder, die durch
      finanzielle oder Sachbeitrage unterstützen. Sie haben die Möglichkeit
      ohne Stimmrecht an der Mitgliederversammlung teilzunehmen.

> 2. Die Kernmitgliedschaft wird auf Vorschlag eines Krenmitgieds durch einen

Reviewer 3: ```suggestion
2. Die Kernmitgliedschaft wird auf Vorschlag eines Kernmitglieds durch einen
```

5. Sie beschließt über Satzungsänderungen und die Auflösung des Vereins.
6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
   schriftlich vorzulegen. Sie beruft eine Person als Kassenprüfer*in, die
   weder dem Vorstand noch einem vom Vorstand berufenen Gremium angehört und
   auch nicht beim Verein angestellt ist. Diese Person überprüft die
   Buchführung einschließlich des Jahresabschlusses und berichtet über das
   Ergebnis vor der Mitgliederversammlung. Sie hat Zugang zu allen Buchungs-

>    und Rechnungsunterlagen des Vereins."

Reviewer 3: ```suggestion
   und Rechnungsunterlagen des Vereins.
```
stray ", or did I miss the opening one?

   kommissarische Vorstandsmitglied bleibt bis zur nächsten
   Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
   Vorstand entscheidet die Mitgliederversammlung.
6. Der Vorstand kann zur Führung der laufenden Geschäfte eine Person als
   Geschäftsführer*in bestellen. Diese nimmt dann an den

>    Mitgliederversammlungen und Vorstandssitzungen teil."

Reviewer 3: same here.
```suggestion
   Mitgliederversammlungen und Vorstandssitzungen teil.
```

## 10. Protokolle

1. Alle Beschlüsse des Vorstandes und der Mitgliederversammlung werden in einem
   elektronisch übersetzbaren Format in deutscher Sprache dokumentiert und
   stehen den Mitgliedern zur Einsichtnahme und zur Prüfung der Beschlüsse
   durch das Registergericht zur Verfügung.
2. Protokolle der Mitgliederversammlung werden vom Versammlungsleiter und vom

>    Protokollführer unterschieben.

Reviewer 3: ```suggestion
   Protokollführer unterschrieben.
```

   besteht auch über das Ausscheiden der oben genannten Personen aus dem Verein
   hinaus.
3. Der Verein wahrt das Briefgeheimnis für den gesamten Schriftverkehr gemäß

>    Artikel 10 des en Grundgesetzes und verpflichtet sich zur Einhaltung

Reviewer 3: ?

## 1. Association Name, Registered Office, Financial Year

1. The Association is registered as Mixxx e.V.

> 2. The registered offices are located in .... . If no fixed office is

Reviewer 5: let's make sure we don't forget this
```suggestion
2. The registered offices are located in **TODO FILL IN LOCATION** . If no fixed office is
```

   2. **Extraordinary members**: Natural persons and legal bodies supporting
      the Association’s aims and are allowed to participate in the General
      Assemblies, but have no voting rights and np obligations to cooperate.
      This gives members the opportunity to pause their cooperation and
      contribution to Mixxx without necessarily losing their membership.
   3. **Supporting Members**: Extraordinary members primarily supporting
      through financial or material contributions. They have the opportunity to
      participate in the General Assemblies without voting rights.
2. Core membership is granted upon the proposal of a core member through a

>    resolution by a General Assemblie or via an Internet Vote with a simple

Reviewer 3: ```suggestion
   resolution by a General Assembly or via an Internet Vote with a simple
```

## 14. Data Protection in the Association

> 1. In order to fulfil the purposes and tasks of the association, personal data

Reviewer 5: At least in American English, "fulfill" has two ls at the end

```suggestion
1. In order to fulfill the purposes and tasks of the association, personal data
```

## 10. Meeting Minutes

> All decisions by the Board and General Assembly are documented in an in an 

Reviewer 3: ```suggestion
All decisions by the Board and General Assembly are documented in an
```
